include/etl/fsm.h

```diff
@@ -216,6 +223,12 @@ namespace etl
     {
     }
 
+	//*******************************************
+	/// State ID to indicate "staying" in a state vs. re-entering it.
+	//	Messages can return this as a value to indicate no state transition.
+	//*******************************************
+    static etl::fsm_state_id_t stay() { return -1; }
```

Comment:
Isn't this a duplicate of line 159?

Reply:
ah thanks! i forgot to delete the other one (I had to move `stay` from `fsm` to `fsm_state` due to the order of the classes in the .h file and forgot to clean one up).

i forgot to mention, the `fsm.h` - it's auto generated from `fsm_generator.h`, so you only need to review the one since they are generated from the other.